app/models/post.rb

```diff
@@ -2,6 +2,9 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   has_many :comments
   has_many :likes
+  validates :title, presence: true, length: { maximum: 250, too_long: '%<count>s characters is the maximum allowed' }
+
```

Comment:
- [ ] As it is a requirement for this project, please kindly add the following  validation rules to your `post model`. 
  - CommentsCounter must be an integer greater than or equal to zero.
  - LikesCounter must be an integer greater than or equal to zero. 

Please kindly ensure all custom methods in the models should have a corresponding unit test.